src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.ts

```diff
@@ -0,0 +1,162 @@
+import { BaseComponent } from './../../../../view/components/_common/_base/base-component';
+import { Component, OnInit, OnDestroy, Output, EventEmitter, ViewChild, ElementRef } from '@angular/core';
+import { FieldRow } from '../shared';
+import { HotkeysService, Hotkey } from 'angular2-hotkeys';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+import { MessageService } from './../../../../api/services/messages-service';
+
+/**
+ * Display select columns row
+ *
+ * @export
+ * @class ContentTypeFieldsAddRowComponent
+ */
+@Component({
+    selector: 'content-type-fields-add-row',
```

Comment:
Prefix the selector with ```dot``` and rename it like ```dot-add-rows``` because we'll be using this in edit pages and not only content types... in fact we probably move it some place else in the near future.